docusaurus/docs/Android/02-client/06-guides/01-push-notifications/04-xiaomi.mdx

```diff
@@ -51,7 +51,7 @@ dependencies {
 Then, create a `XiaomiPushDeviceGenerator` and add it to the list of generators in `NotificationConfig`, which you should pass into `ChatClient.Builder` when you initialize the SDK:
 
 ```kotlin {3-7,11}
-val notificationConfig = NotificationConfig(
+val notificationsConfig = NotificationConfig(
```

Comment:
Didn't fully get an idea of renaming `notificationConfig` intro the plural version while the class name is still `NotificationConfig`